src/storage/v2/vertex_accessor.cpp

```diff
@@ -335,25 +338,29 @@ Result<std::vector<std::tuple<PropertyId, PropertyValue, PropertyValue>>> Vertex
 
   if (vertex_->deleted) return Error::DELETED_OBJECT;
 
+  bool should_apply_delta_if_identical_update = storage_->config_.salient.items.delta_on_identical_property_update;
```

Comment:
I like it when we make boolean expressions more concise by naming them using variable. If this variable is not getting changed can we make this `const`?